custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/connectorconfiguration/auth/OAuth2Defaults.java

```diff
@@ -60,4 +60,13 @@
      * OAuth2 Grant types supported by connector.
      */
     List<OAuth2GrantType> oAuth2GrantTypesSupported();
+
+    /**
+     * OAuth2 custom parameters needed by the connector.
+     *
+     * AppFlow uses clientId, scope default parameters for required generating the auth code, Users don't need to pass
```

Comment:
AppFlow have clientId and scope defined as the default parameter for AUTH_URL to generate the Authorization code in case of 3 LO. Connector developer don't have to define clientId and scope as OAuth2Custom Parameter for AUTH_URL.